.lune/util/generator/generator.luau

```diff
@@ -0,0 +1,51 @@
+local fs = require("@lune/fs")
+local net = require("@lune/net")
+
+local Generator = {}
+
+function Generator.removeLegacyFiles()
+	if fs.isDir("docs/classes") then
+		fs.removeDir("docs/classes")
+	end
+
+	fs.writeDir("docs/classes")
+end
+
+function Generator.writeClassName(className: string)
+	local classPath = string.split(className, ".")
+	local fullPath = "docs/classes/"
+
+	local fileName = table.remove(classPath, #classPath) :: string
+	local netSafeFileName = net.urlEncode(fileName, false)
+
+	for index, path in classPath do
+		if not fs.isDir(fullPath .. path) then
+			fs.writeDir(fullPath .. path)
+		end
+
+		fullPath ..= `{path}/`
+	end
+
+	fs.writeFile(fullPath .. `{netSafeFileName}.md`, ``)
+	return fullPath .. `{netSafeFileName}.md`
+end
```

Comment:
Couple things:
- We don't need to URL encode file names here, not sure why that's being done
- The docs directory should be flat, with all markdown files at the root for it to work with pesde's doc system

Reply:
> We don't need to URL encode file names here, not sure why that's being done

the path is rendered in the URL bar, w/o url-encoding the filenames you won't see what the path looks like, which can create issues when trying to like to another page.

> The docs directory should be flat, with all markdown files at the root for it to work with pesde's doc system

ok